rllib/agents/pg/pg_torch_policy.py

```diff
@@ -1,15 +1,36 @@
+from typing import Dict, List, Optional, Type, Union
+
 import ray
-from ray.rllib.agents.pg.pg_tf_policy import post_process_advantages
+from ray.rllib.agents.pg.utils import post_process_advantages
 from ray.rllib.evaluation.postprocessing import Postprocessing
+from ray.rllib.models.torch.torch_action_dist import TorchDistributionWrapper
+from ray.rllib.models.modelv2 import ModelV2
+from ray.rllib.policy import Policy
 from ray.rllib.policy.sample_batch import SampleBatch
 from ray.rllib.policy.torch_policy_template import build_torch_policy
 from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.types import TensorType
 
 torch, _ = try_import_torch()
 
 
-def pg_torch_loss(policy, model, dist_class, train_batch):
-    """The basic policy gradients loss."""
+def pg_torch_loss(
```

Comment:
This is framework-agnostic, so it shouldn't be located in any of the two policy files.